am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -27,17 +29,17 @@ private TestDataFactory() {
 
     public static ExplicitAccessGrant createGrantForWholeDocument(String resourceId,
                                                                   Set<Permission> permissions) {
-        return createGrantForWholeDocument(resourceId, ACCESSOR_ID, permissions);
+        return createGrantForWholeDocument(resourceId, ACCESSOR_IDS, permissions);
     }
 
     public static ExplicitAccessGrant createGrantForWholeDocument(String resourceId,
-                                                                  String accessorId,
+                                                                  List<String> accessorId,
```

Comment:
Why to change it? Can't we have overloaded method that accepts single user ID?

Reply:
👍